cognite/client/data_classes/filters.py

```diff
@@ -89,7 +90,10 @@ def dump(self, camel_case_property: bool = False) -> dict[str, Any]:
 
     @classmethod
     def load(cls, filter_: dict[str, Any]) -> Filter:
-        (filter_name,) = filter_
+        if len(filter_) > 1:
+            warnings.warn(f"Filter {filter_} has more than one key, only the first key will be used", stacklevel=2)
+
+        filter_name = next(iter(filter_))
```

Comment:
This won't necessarily pick the right key. See how it was fixed for `Aggregation` here https://github.com/cognitedata/cognite-sdk-python/commit/43e4d2883e0b90aa5f3f304086a68d815f1528e8

Reply:
Good point, fixed.